hub/api/dataset.py

```diff
@@ -144,9 +146,10 @@ def create_tensor(
                 For example, `htype="image"` would have `dtype` default to `uint8`.
                 These defaults can be overridden by explicitly passing any of the other parameters to this function.
                 May also modify the defaults for other parameters.
-            chunk_size (int): Optionally override this tensor's `chunk_size`. In short, `chunk_size` determines the size of files (chunks) being created to represent this tensor's samples.
-                For more on chunking, check out `hub.core.chunk_engine.chunker`.
-            dtype (str): Optionally override this tensor's `dtype`. All subsequent samples are required to have this `dtype`.
+            chunk_size (int, optional): The target size for chunks in this tensor.
+            dtype (str, optional): The data type to use for this tensor.
+                Will be overwritten when the first sample is added.
+            compression (str, optional): Compressor name to apply on the tensor.
```

Comment:
```suggestion

            default_compression (str, optional): Optionally override this tensor's `default_compression`. If a sample is already compressed during ingestion (when using the `hub.util.load.read` method), this `default_compression` is ignored. However if a sample is added that is uncompressed, `default_compression` will be used.
```

also alongside this change, can you rename `compression` to `default_compression`?